### ISSUE TITLE
Proposal: LLVM .deb packages

### ISSUE THREAD

**Issue by leandron** [WorksOnArm/cluster#172]
### Name, email, company, job title

Leandro Nunes, Arm, Software Engineer

### Project Title and description

LLVM Debian packages

### Which members of the community would benefit from your work?
It will benefit users interested in recent LLVM builds, usually to support other OSS projects that rely on cutting edge features.

### Is the code that you’re going to run 100% open source? 

Yes, https://github.com/llvm.

### What infrastructure (computing resources and network access) do you need? (see: https://www.packet.net/bare-metal/)?

Bare metal AArch64 machine (no need for AArch32).

### Describe the continuous integration (CI) system in use or desired for this project.

All is to be build natively on AArch64 worker machines to be added to https://llvm-jenkins.debian.net/.

### Describe the monitoring system in use or desired for this project.

Yes, we would be very interested to install Prometheus, capture machine usage and share the data.

### Please state your contributions to the open source community and any other relevant initiatives

Other contributions using Works on Arm include CI machines (per commit) for LLVM and support for CI on Flang (Fortran compiler).

**Comment by vielmetti** [comment]
Attn @cayu53

**Comment by cayu53** [comment]
Happy to support. 

Would be good for Leandro (or community) to raise visibility (either blog post or other suitable medium) once this is in place/running.

**Comment by vielmetti** [comment]
Noting for reference that previous LLVM development work was made visible with this blog post from @leandron in January 2019.

https://community.arm.com/developer/tools-software/tools/b/tools-software-ides-blog/posts/accelerating-open-source-llvm-development

**Comment by vielmetti** [comment]
Reopening and removing `wontfix`, with the intent to resolve this week (week/39)

**Comment by vielmetti** [comment]
This has been set up, and is added to the "Sponsored - Arm - LLVM" project to reflect the ongoing organization and supervision of the work.

**Comment by leandron** [comment]
Thanks Ed @vielmetti
